lib/SILOptimizer/IPO/GlobalOpt.cpp

```diff
@@ -284,7 +284,7 @@ bool SILGlobalOpt::isInLoop(SILBasicBlock *CurBB) {
 
   if (LoopCheckedFunctions.insert(F).second) {
     for (auto I = scc_begin(F); !I.isAtEnd(); ++I) {
-      if (I.hasLoop())
+      if (I.hasCycle())
```

Comment:
Was the function just renamed?

Reply:
Yes. The commit that changes this points to the commit in LLVM that renamed it.